### ISSUE TITLE
Object.defineProperties error when using webpack-dev-server

### ISSUE THREAD

**Issue by odoe** [OpenNTF/dojo-webpack-plugin#151]
I'm using webpack-dev-server 2.11.1 and webpack 3.11.0.

With version 2.6.0 of the dojo-webpack-plugin, I can build fine with webpack, but if I use webpack-dev-server, I get the following error.

```
TypeError: Cannot redefine property: compilation
    at Function.defineProperties (<anonymous>)
    at Compiler.compiler.plugin (/Users/rene8209/Sites/demos/arcgis-webpack-plugin-sample/node_modules/dojo-webpack-plugin/lib/DojoAMDModuleFactoryPlugin.js:49:12)
```

Looks like this recent change here is causing some issue with dev-server.
https://github.com/OpenNTF/dojo-webpack-plugin/blob/master/lib/DojoAMDModuleFactoryPlugin.js#L49-L52

```
  Object.defineProperties(context, {
    compilation: {value: compilation},
    params: {value: params}
  });
```

**Comment by odoe** [comment]
Just tested in my local node_modules install and I changed the above to:

```
context.compilation = compilation;
context.params = params;
```

And that seems to work without issues so far.

**Comment by chuckdumont** [comment]
Fixed in build 2.6.1

**Comment by odoe** [comment]
Works, thanks!
